core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -9,26 +9,45 @@ use {
     std::time::{Duration, Instant},
 };
 
-const STATS_INTERVAL_SECONDS: u64 = 10; // Log stats every 10 second
+const STATS_INTERVAL_SECONDS: u64 = 1; // Log stats every second
```

Comment:
Could make this of type Duration for convenience: `Duration::from_secs(1)`

Reply:
Done.